scripts/tools/pad_number_generators.py

```diff
@@ -6,7 +6,8 @@
 
 Available generators:
 ----------------------
-- increment: Is a simple generator that increments the pin number by one.
+- increment: Generates pin numbers increasing by one and also supports skipping
+            pin numbers (skipped pins return a bogus value of -1)
```

Comment:
This should say that skipping pins is based on the `deleted_pins` keywordargs

Reply:
Alright. Updated.